src/pages/settings/Wallet/WalletSection.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Section from '../../../components/Section';
+
+const propTypes = {
+    icon: PropTypes.func.isRequired,
+    children: PropTypes.node.isRequired,
+    subtitle: PropTypes.string.isRequired,
+    title: PropTypes.string.isRequired,
+};
+
+function WalletSection({icon, children, subtitle, title}) {
+    return (
+        <Section
+            icon={icon}
+            subtitle={subtitle}
+            title={title}
+        >
+            {children}
+        </Section>
+    );
+}
+
+WalletSection.displayName = 'WalletSection';
+WalletSection.propTypes = propTypes;
+
+export default WalletSection;
```

Comment:
This component seems redundant, do we really need it?

Reply:
Not sure, it was Pedro's decision to add it, maybe let's wait until tomorrow, so he can share his opinion on that, wdyt?

Reply:
@thiagobrez 
It was requested on the technical document

Reply:
Where is this being used in this PR?

Reply:
@grgia I was using it in the new WalletPage from the [first PR](https://github.com/Expensify/App/pull/26406), but now it's removed 😄